traffic/algorithms/navigation.py

```diff
@@ -710,7 +716,11 @@ def go_around(
         if airport is None:
             return None
 
-        first_attempt = next(self.aligned_on_ils(airport), None)
+        # you need to be aligned at least twice on a rway to have a GA:
+        if len(self.aligned_on_ils(airport, **kwargs)) < 2:
```

Comment:
So we run aligned_on_ils twice now! Can't we store intermediate results for performance?